src/components/EntranceFees/EntranceFees.js

```diff
@@ -1,29 +1,29 @@
-import './EntranceFees.css';
-import EntranceFeeCard from '../EntranceFeeCard/EntranceFeeCard';
-import PropTypes from 'prop-types';
+// import './EntranceFees.css';
+// import EntranceFeeCard from '../EntranceFeeCard/EntranceFeeCard';
+// import PropTypes from 'prop-types';
 
-export default function EntranceFees({parks}) {
+// export default function EntranceFees({parks}) {
 
-  const fees = parks[0].entranceFees.map(fee => {
-      return (
-        <section>
-            <EntranceFeeCard
-            title={fee.title}
-            cost={fee.cost}
-            description={fee.description}
-            />
-        </section>
+//   const fees = parks[0].entranceFees.map(fee => {
+//       return (
+//         <section>
+//             <EntranceFeeCard
+//             title={fee.title}
+//             cost={fee.cost}
+//             description={fee.description}
+//             />
```

Comment:
Love the use of semantics, I don't know what this element visually appears to be, but looking at the content an article tag might be fitting